conf/denas/asr/e2eaiok_denas_asr_pruner.conf

```diff
@@ -0,0 +1,33 @@
+# conf for transformer based asr with pruner
```

Comment:
please add comments on key configuration parameters and make sure it will be reflected in reame.